invenio_app_rdm/theme/templates/themes/default/invenio_app_rdm/site_footer.html

```diff
@@ -6,29 +6,24 @@
     under the terms of the MIT License; see LICENSE file for more details.
 #}
 
-<div class="footer-bottom sub-footer theme-default">
+<div class="footer-bottom sub-footer theme-default rel-p-1">
   <div class="ui container">
     <div class="ui grid">
       {%- block footer_bottom_left %}
 
         <div
           class="eight wide computer six wide tablet sixteen wide mobile column left middle aligned"
         >
-          {% trans invenio_rdm="http://inveniosoftware.org/products/rdm" %}Powered by
-            <a href="{{ invenio_rdm }}">InvenioRDM</a>{% endtrans %}
+          {% trans cern_data_center="http://information-technology.web.cern.ch/about/computer-centre" %}Powered by
```

Comment:
minor: I thin this shouldn't be in `app-rdm`, since not all InvenioRDM instances are hosted at the CERN ata Center :)